docs/quickstarts/integrations-slack-notifs-qs/integrations-slack-notifs-qs.yml

```diff
@@ -28,9 +28,9 @@ spec:
   introduction: |-
     Use Hybrid Cloud Console notifications to learn about events that might impact your organization. 
     
-    You can receive notifications in the console, or configure the console to send event notifications to a third-party application such as Slack. In Slack, you can send notifications to all users on a channel, or a selected user.
-
-    Configure a Slack channel to receive event notifications when a new User Access group is created in the console. To validate your configuration, you’ll also send a test event.
+    You can receive notifications in the console, or configure the console to send event notifications to a third-party application such as Slack. 
+    
+    In Slack, you can send event notifications to all users on a channel, or to a selected user.
```

Comment:
I don't think "selected user" is supported yet, unless that became possible after the recent change to the Slack URLs. I'll check and confirm.

Reply:
@gwenneg I tested this by sending messages only to myself (it worked) - perhaps I need to reword this to make it clearer. Let me know what you find with testing too. What do you think would make this better?

Reply:
Sorry, I wasn't aware Slack users could receive notifications as private message. The wording looks perfectly fine to me.

Reply:
No problem - thanks for checking this for me, it's great to have more eyes on the content!